### ISSUE TITLE
Issue density not updating

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#967]
<img src="https://uploads.linear.app/e79c939f-ca5d-4ce4-8c85-8f93e29766e1/6bf087bf-fc49-4f94-9bdb-a09711b096ab/aad10ef0-ccb6-4053-8c1f-ed56c5684ae7?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiL2U3OWM5MzlmLWNhNWQtNGNlNC04Yzg1LThmOTNlMjk3NjZlMS82YmYwODdiZi1mYzQ5LTRmOTQtOWJkYi1hMDk3MTFiMDk2YWIvYWFkMTBlZjAtY2NiNi00MDUzLThjMWYtZWQ1NmM1Njg0YWU3IiwiaWF0IjoxNzQ2NzMzODMwLCJleHAiOjMzMzE3MjkzODMwfQ.rDGADrDyoKlViL2VRfIJJMPzJBofPIaIfVcO-_3arDk " alt="Screenshot 2025-05-08 at 3.42.18 PM.png" width="838" data-linear-height="645" />

`_edac_issue_density` doesn't seem to be updating. Ensure `_edac_issue_data` is formatted correctly and should have multiple values.
